Modules/Filtering/ImageGrid/include/itkResampleImageFilter.h

```diff
@@ -169,7 +169,7 @@ class ITK_TEMPLATE_EXPORT ResampleImageFilter :
 
   /** Input pixel continuous index typdef */
   using ContinuousInputIndexType =
-      ContinuousIndex< TTransformPrecisionType, ImageDimension >;
+      ContinuousIndex< TInterpolatorPrecisionType, ImageDimension >;
```

Comment:
Just checked: this change does indeed fix compilation errors when TInterpolatorPrecisionType != TTransformPrecisionType.

Reply:
The driving problem is replacing the WarpImageFilter with this one, and ran into this issue.